cime_config/testlist_allactive.xml

```diff
@@ -1,425 +1,127 @@
 <?xml version="1.0"?>
 <testlist version="2.0">
-  <test name="ERI" grid="f09_g17" compset="B1850" testmods="allactive/defaultio_noclminterp">
+
+  <test name="ERS_Ld5" grid="f09_tn14" compset="N1850frc2" testmods="allactive/defaultio">
     <machines>
-      <machine name="bluewaters" compiler="pgi" category="prebeta"/>
+      <machine name="betzy" compiler="intel" category="noresm_prealpha"/>
```

Comment:
bluewaters? Man, that's a blast from the past (good thing this list was *almost* up to date :)

Reply:
I thought I got rid of all of this. 
And yes - I ran the full set of all active regression tests yesterday and they passed!

Reply:
Let me get rid of bluewaters.

Reply:
> Let me get rid of bluewaters.

Sorry, that was just an observation of where the testlist started when you made your changes. A quick check shows that betzy is the only machine listed in the PR version of this file.